### DISCUSSION TITLE
Error on large custome model

### DISCUSSION THREAD

**Discussion by LunarBuddies** [Purfview/whisper-standalone-win#260] [Q&A] [answered]
Hi

I get this error on a custome large model. The same model in small and medium works fine.
Any ide what is wrong here:

Invalid input features shape: expected an input with shape (1, 128, 3000), but got an input with shape (1, 80, 3000) instead
 [17324] Failed to execute script '__main__' due to unhandled exception!

the model is
https://huggingface.co/NbAiLab/nb-whisper-large

tried both [Faster-Whisper-XXL r192.3.4 and 192.3

**Comment by Purfview** [comment, marked as answer]
There are missing `preprocessor_config.json` and `tokenizer.json` files.
Try to get them from there: https://huggingface.co/Systran/faster-whisper-large-v3/tree/main

How is performance of this model?

**Reply by LunarBuddies** [reply]
Thx, not sure of the performance yet. it is trained on Norwegian.

**Reply by Purfview** [reply]
Report if it performs better on Norwegian than original models.
Usually I see that people have problems with the timestamps with those fine-tuned models.

**Reply by iTula** [reply]
The same situation with the finetuned chinese model:https://hf-mirror.com/BELLE-2/Belle-whisper-large-v3-zh
The Faster-Whisper-XXL r192.3.4 won't recognize the finetuned model.

**Reply by Purfview** [reply]
> The same situation with the finetuned chinese model:https://hf-mirror.com/BELLE-2/Belle-whisper-large-v3-zh
The Faster-Whisper-XXL r192.3.4 won't recognize the finetuned model.

This model is not faster-whisper [ctranslate2] model. It's for Whisper-OpenAI.
